### DISCUSSION TITLE
Unable to save circuit coordinates

### DISCUSSION THREAD

**Discussion by ShadowCipher09** [mattieserver/netbox-topology-views#413] [Q&A]
### NetBox version

3.6

### Topology Views version

v.3.8.0

### Steps to Reproduce

I have the option to save coordinates enabled when I move objects, but in circuits it constantly returns to its place

### Expected Behavior

That the coordinates are saved correctly

### Observed Behavior

Back to its place

**Comment by kentra** [comment]
I'm having the same problem.
I have activated "allow_coordinates_saving" and "always_save_coordinates".

When following the README "Convert custom field to Coordinate Groups" coordinates are not being saved.
In dev tools (chrome) I can see that I get a bunch of HTTP 500 from "/save-coords/"


![Screenshot 2023-11-03 100149](https://github.com/mattieserver/netbox-topology-views/assets/20362840/6cb99e26-2927-467f-a6b2-44babf07ed20)

**Reply by kentra** [reply]
Further investigations reveal that "group" is missing in payload in JS function.

![Screenshot 2023-11-03 105911](https://github.com/mattieserver/netbox-topology-views/assets/20362840/652e51ce-c335-4c85-8845-1c081044e9ef)

**Reply by kkthxbye-code** [reply]
You probably either forgot to run `collectstatic` or your browser is caching the javascript file.

**Comment by dreng** [comment]
Does the Javascript console show any errors?
Are you using a custom base path?

@kentra You didn't write which versions of NetBox and NTV you use. I suggest that you update both first if it's not up to date.
